### DISCUSSION TITLE
请问下第三方的filecoin主链数据源有哪些选择？

### DISCUSSION THREAD

**Discussion by hzysvilla** [filecoin-project/community-china#65] [应用开发]
我需要币价，算力，区块高度等信息，但infura只提供了算力信息，filfox这些区块链似乎又没有公开的api文档或者付费的api服务。

**Comment by bary321** [comment]
可能需要自己同步一个节点去抓数据了

**Reply by hzysvilla** [reply]
你应该抓不到price信息那些吧？还是需要一个数据源。

**Comment by Fatman13** [comment]
Coingecko，CMC应该有价格。
